cypress/integration/ui/dashboard_filter/dashboard_filter.js

```diff
@@ -0,0 +1,26 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const OPTIONS = { timeout: 15000 }
+
+When('I add a {string} filter', () => {
+    cy.contains('Add filter').click()
+
+    cy.get('[data-test="undefined-button-pe"]').click()
+
+    cy.get('[data-test="dhis2-uicore-transfer-sourceoptions"]')
+        .contains('Last 6 months')
```

Comment:
For scalability I would move this out to a const e.g. `TEST_PERIOD` to be used both for the action and assertion below.

Reply:
👍